_posts/2022-05-02-the-na-bug/references.bib

```diff
@@ -0,0 +1,40 @@
+@article{cohen_problems_2021,
```

Comment:
I exported this from Zotero and removed entries that weren't cited.

One interesting thing to note: the data generated by Zotero included a `file` field. I removed that since it links to a local copy of the paper. Not sure if it would have caused any issues with rendering...